irobot_events_executor/include/rclcpp/timers_manager.hpp

```diff
@@ -150,10 +143,11 @@ class TimersManager
   std::chrono::nanoseconds get_head_timeout();
 
   /**
-   * @brief Function to check if the timers thread is running
-   * @return true if timers thread is running
+   * @brief Implements a loop that keeps executing ready timers.
+   * This function is intended to run in its own thread.
```

Comment:
I don't think that this comment is useful.
Nothing prevents someone to create a timer manager and call this function, without any thread.
We should simply mention that it's a Blocking function